.vsts-ci/templates/releaseBuildPhase.yml

```diff
@@ -63,6 +83,16 @@ jobs:
     displayName: Build snap $(channel)
     condition: and(succeeded(), ne(variables['$(channel)'],'no'))
 
+  - powershell: |
+        sudo snap install $(PS_SNAP_PATH) --classic --dangerous
```

Comment:
why the dangerous switch here? Does this mean we need to error handle in some way due to it?

Reply:
`--dangerous` is a flag to indicate to snapd that you are accepting that you are installing a potentially dangerous download - it's intended to discourage users from downloading random `.snap` files off the internet instead of using the Snap Store. In this instance it's because we're installing the snap that we just built so it's merely boilerplate to allow us to do the installation.

Reply:
thank you for this explanation @diddledani  :D